samples/tab-meeting-context/nodejs/AppPackage/manifest.json

```diff
@@ -0,0 +1,62 @@
+{
+  "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.12/MicrosoftTeams.schema.json",
+ "manifestVersion": "1.12",
+  "version": "1.0.1",
+  "id": "{{Manifest-id}}",
+  "packageName": "com.teams.meetingtabcontext",
+  "developer": {
+    "name": "Microsoft",
+    "websiteUrl": "https://www.microsoft.com",
+    "privacyUrl": "https://www.teams.com/privacy",
+    "termsOfUseUrl": "https://www.teams.com/termsofuser"
+  },
+  "icons": {
+    "color": "color.png",
+    "outline": "outline.png"
+  },
+  "name": {
+    "short": "Meeting Tab context",
+    "full": "This sample shows the contents of tab context object in a meeting."
+  },
+  "description": {
+    "short": "Tab context for private/shared channels",
+    "full": "This sample shows the contents of tab context object in a private and shared channel."
+  },
+  "accentColor": "#5558AF",
+  "configurableTabs": [
+    {
+      "configurationUrl": "https://{{Domain Name}}/configure",
+      "canUpdateConfiguration": true,
+      "scopes": [
+        "team",
+        "groupchat"
+      ],
+      "context": [
+        "meetingChatTab",
+        "meetingDetailsTab"
+      ]
+    }
+  ],
+  "permissions": [
+    "identity",
+    "messageTeamMembers"
+  ],
+  "validDomains": [
+    "*.ngrok.io",
+    "<<Domain Name>>"
+  ],
+    "webApplicationInfo": {
+    "id": "<<App id>>",
+    "resource": "https://RscBasedStoreApp"
```

Comment:
Is this predefined URL?

Reply:
Do we need to always use this or it can change?
If it can change, add comment

Reply:
We are using this predefined URL in all samples which does not change